app/services/quote_service.rb

```diff
@@ -0,0 +1,10 @@
+class QuoteService
+  def self.quote_search(keyword)
+    response = conn.get("search/quotes?query=#{keyword}&limit=10")
+    output = JSON.parse(response.body, symbolize_names: :true)[:results]
+  end
+
+  def self.conn
+  Faraday.new(url: 'https://api.quotable.io/')
+  end
+end
```

Comment:
👍 This all looks good. The only thing that I would refactor about this is that on line 4, you are filtering the JSON response. There are still things outside of `results` that we need. If you take a look back at the user story, we want to see the total count of quotes that are found. That value is in a key labeled `totalCount` and it is outside of the `results` hash. So, in your service, let it just get ALL the data. Then, in your facade, let it pick and choose what exactly it wants from the payload.